src/main/resources/synthea.properties

```diff
@@ -225,16 +227,22 @@ generate.providers.minimum = 1
 generate.providers.maximum_search_distance = 32
 
 # Payers
+generate.payers.insurance_plans.default_file = payers/insurance_plans.csv
+generate.payers.insurance_plans.eligibilities_file = payers/insurance_eligibilities.csv
 generate.payers.insurance_companies.default_file = payers/insurance_companies.csv
+generate.payers.insurance_companies.medicaid_eligibility = payers/medicaid_eligibility.csv
 generate.payers.insurance_companies.medicare = Medicare
+]
 generate.payers.insurance_companies.medicaid = Medicaid
 generate.payers.insurance_companies.dual_eligible = Dual Eligible
-
+# The percentage of a person's income that they are willing to spend on health insurance premiums.
+generate.payers.insurance_plans.income_premium_ratio = 0.13
+# The chance of rejection 
 # Payer selection behavior
 # How patients select a payer:
 #  best_rates - select insurance with best rates for person's existing conditions and medical needs
 #  random  - select randomly.
```

Comment:
The comments list two options: `best_rates` and `random`, yet you've changed the default value to `gov_priority`.

Please update the comments to provide information to our users on the new options they have available.